From 85b80e70a92ad66b86dc0a1636c59145b9fbf39c Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 26 Feb 2013 16:10:54 +0800
Subject: [PATCH 2396/2508] ENGR00251849-2 ldo_bypass:print the ldo_bypass
 mode directly, not obscure number

You can see "cpu regulator mode:ldo_bypass" log directly from console when boot.
not "cpu regulator init ldo=1" before.
Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/mach-mx6/cpu_regulator-mx6.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_regulator-mx6.c b/arch/arm/mach-mx6/cpu_regulator-mx6.c
index 59bc383..0e91c8c 100644
--- a/arch/arm/mach-mx6/cpu_regulator-mx6.c
+++ b/arch/arm/mach-mx6/cpu_regulator-mx6.c
@@ -85,7 +85,8 @@ void mx6_cpu_regulator_init(void)
 		soc_reg_id = "cpu_vddsoc";
 		pu_reg_id = "cpu_vddgpu";
 	}
-	printk(KERN_INFO "cpu regulator init ldo=%x\n", enable_ldo_mode);
+	printk(KERN_INFO "cpu regulator mode:%s\n", (enable_ldo_mode ==
+		LDO_MODE_BYPASSED) ? "ldo_bypass" : "ldo_enable");
 	cpu_regulator = regulator_get(NULL, gp_reg_id);
 	if (IS_ERR(cpu_regulator))
 		printk(KERN_ERR "%s: failed to get cpu regulator\n", __func__);
-- 
1.7.7.4

